nautobot/core/views/generic.py

```diff
@@ -322,6 +323,15 @@ def get(self, request):
             }
             RequestConfig(request, paginate).configure(table)
             table_config_form = TableConfigForm(table=table)
+            per_page = request.GET.get("per_page") or request.user.get_config(
+                "pagination.per_page", get_settings_or_config("PAGINATE_COUNT")
+            )
+            max_page_size = get_settings_or_config("MAX_PAGE_SIZE")
+            if int(per_page) > int(max_page_size):
+                messages.error(
```

Comment:
I think `error` is overly strong here.

```suggestion
                messages.warning(
```